x/wasm/keeper/addresses.go

```diff
@@ -22,8 +22,8 @@ func (k Keeper) ClassicAddressGenerator() AddressGenerator {
 	}
 }
 
-// PredicableAddressGenerator generates a predictable contract address
-func PredicableAddressGenerator(creator sdk.AccAddress, salt, msg []byte, fixMsg bool) AddressGenerator {
+// PredictableAddressGenerator generates a predictable contract address
+func PredictableAddressGenerator(creator sdk.AccAddress, salt, msg []byte, fixMsg bool) AddressGenerator {
```

Comment:
Is this a breaking change @pinosu?

Reply:
No, it's not a breaking change. Looks good! 💯